### ISSUE TITLE
NOT Issue: -seed option

### ISSUE THREAD

**Issue by Allycen** [jcjohnson/neural-style#275]
Could someone explain what -seed option do? Can't figure it out. Result pictures tend to be almost the same

**Comment by jcjohnson** [comment]
The `-seed` option sets the initial state of the random number generator, which is used to generate the initial random image. If you don't explicitly pass a seed then you can get slightly different results between runs even when your style image, content image, and all other parameters are identical; passing a seed eliminates this randomness.

**Comment by Runescaped** [comment]
> Result pictures tend to be almost the same

Exactly. It allows you to reproduce your results, although there will be some slight differences.

**Comment by Allycen** [comment]
Does it matter if it is a negative value or positive?
